operational_graphics/toolboxes/scripts/PointTargetGRG.py

```diff
@@ -466,11 +471,11 @@ def main():
         #UPDATE
         toolboxPath = None
         if appEnvironment == "ARCGIS_PRO":
-            toolboxPath = os.path.join(os.path.dirname(sysPath), "Clearing Operations Tools.tbx")
+            toolboxPath = os.path.join(os.path.dirname(sysPath), "ClearingOperationsTools.tbx")
         else:
-            toolboxPath = os.path.join(os.path.dirname(sysPath), "Clearing Operations Tools_10.4.tbx")
+            toolboxPath = os.path.join(os.path.dirname(sysPath), "ClearingOperationsTools.tbx")
```

Comment:
Will need to update the toolbox name here: **ClearingOperationsTools_arcmap.tbx**.